custom_components/tuya_local/devices/wf520d_wifi_dual_touch_dimmer

```diff
@@ -0,0 +1,59 @@
+name: WF520D Dual Touch Dimmer
```

Comment:
HA naming convention is for devices and entities to only capitalise the first word of the name unless the words are trademarks or abbreviations that need to be capitalised.

If you can access the product id (from tuya developer portal, or by a network scan using tinytuya), then it is better to use a generic name above like "Dual dimmer" and add a products section with id and name that includes branding.

Reply:
Sure, I've changed it to "Dual dimmer".